<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tool_editor_widget</class>
 <widget class="QWidget" name="Tool_editor_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="tools_listWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="add_pushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="remove_pushButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="tool_configuration_formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="path_label">
           <property name="text">
            <string>Path</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="path_lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="path_browse_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="arguments_label">
           <property name="text">
            <string>Arguments</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="input_label">
           <property name="text">
            <string>Input</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="output_label">
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="errors_label">
           <property name="text">
            <string>Errors</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="activation_label">
           <property name="text">
            <string>Activation</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="arguments_lineEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="input_lineEdit"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="output_comboBox"/>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="errors_comboBox"/>
         </item>
         <item row="5" column="1">
          <widget class="QKeySequenceEdit" name="activation_keySequenceEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>arguments_lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Tool_editor_widget</receiver>
   <slot>update_current_tool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Tool_editor_widget</receiver>
   <slot>update_current_tool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Tool_editor_widget</receiver>
   <slot>update_current_tool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>errors_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Tool_editor_widget</receiver>
   <slot>update_current_tool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>path_lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Tool_editor_widget</receiver>
   <slot>update_current_tool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_tools_list()</slot>
  <slot>update_current_tool()</slot>
 </slots>
</ui>
